test.sh

```diff
@@ -447,6 +449,10 @@ curl -sSfg -u $SK: $HOST/v1/invoices/upcoming?customer=$cus\&subscription_items[
 
 curl -sSfg -u $SK: $HOST/v1/invoices/upcoming?customer=$cus\&subscription=$sub\&subscription_items[0][id]=si_RBrVStcKDimMnp\&subscription_items[0][plan]=basique-annuel\&subscription_proration_date=1504182686\&subscription_tax_percent=20
 
+si=$(curl -sSfg -u $SK: $HOST/v1/subscription_items?subscription=$sub \
+     | grep -oE 'si_\w+')
+[ -n "$si" ]
```

Comment:
In this first commit, `si` is not a subscription item, it's two subscription items.
So Maybe there could be a first `res` that stores them, a check of how many items there is in `res` and then, in the second commit a `si=$(echo "$res" | head -n 1)`.
It would be more readable for future readers of this code.
What do you think?

Reply:
👍 split into proposed steps